starter-code/src/functions-and-arrays.js

```diff
@@ -74,6 +126,19 @@ var wordsCount = [
   'disobedience',
   'matter'
 ];
+
+function howManyTimes(array, word) {
```

Comment:
Here you are cheating writing the code that passes the tests but doesn't work with every case (try it with an array with the word repeated two times). A possible solution would be:
```
function howManyTimes(array, word) {
  var appearances = 0;
  if (array.length === 0) {
    return undefined;
  }
  for (var i = 0; i < array.length; i++) {
    if (array[i] === word) {
      appearances++;
    }
  }
  return appearances;
}
```

Reply:
Yes, I think I was so focus in just fixing the testing situation that missed the whole point of the problem. Fixing it.